src/shared/port-provider.js

```diff
@@ -246,6 +221,35 @@ export class PortProvider {
     );
   }
 
+  /**
+   * Check that data and origin matches the expected values.
+   *
+   * @param {object} options
+   *   @param {Message} options.allowedMessage - the `data` must match this
+   *     `Message`.
+   *   @param {string} options.allowedOrigin - the `origin` must match this
+   *     value. If `allowedOrigin` is '*', the origin is ignored.
+   *   @param {any} options.data - the data to be compared with `allowedMessage`.
+   *   @param {string} options.origin - the origin to be compared with
+   *     `allowedOrigin`.
```

Comment:
Given that this is an internal method that is very short and simple, I think it would be fine to remove the descriptions here and just specify the types.

Reply:
👍🏼